AnnoyancesFilter/sections/popups.txt

```diff
@@ -92,6 +92,10 @@ sodapdf.com##.be-tooltip
 !
 senpa.io###new-panel
 ||macmost.com/wp-content/themes/*/awpop-subscribe-*.js
+e.ibe.kr#$##ouibounce-modal { display: none!important; }
+e.ibe.kr#$#.modal-backdrop { display: none!important; }
+e.ibe.kr#$#body { overflow: visible!important; }
+e.ibe.kr###modal
```

Comment:
Is it required? It is with `display: none;` on my end.

Reply:
![Screenshot 2020-07-15 at 13 07 24](https://user-images.githubusercontent.com/18536328/87532728-40a56380-c69c-11ea-92db-00ed33c7aae2.png)